projects/core/src/lib/time/time-step-selector/time-step-selector.component.html

```diff
@@ -11,3 +11,7 @@
     ></mat-slider>
     <span>{{ currentTimeFormatted | async }}</span>
 </div>
+
+<div *ngIf="!displayTimeLine()" class="padded container mat-typography" fxLayout="row" fxLayoutAlign="space-between center">
```

Comment:
Please try to make it as high as with the slider.

Reply:
I found that with a padding of `0.85rem` and margin of `0 0.2rem` it's practically identical to how it looks with the slider. I guess these are oddly specific values, but I can't figure out a nicer way to emulate the padding/margin that happens due to the slider, when it is there. d64c6de9ffb5fdb85f1f9cb85c7612b7e4b260dd